src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -246,7 +246,9 @@ public function updateCombinationFromListingAction(int $combinationId, Request $
             );
         }
 
-        return $this->json([]);
+        return $this->json([
+            'message' => $this->trans('Update successful', 'Admin.Notifications.Success'),
```

Comment:
```suggestion
            'message' => $this->trans('Successful update.', 'Admin.Notifications.Success'),
```

This is the wording that is used in the product controller after recent wording changes

Reply:
@jolelievre 
We should keep "Update successful" :) Every time I see "Successful update." in a PR I try to replace it with "Update successful" :)

Reply:
Ok then it's in the Product page that we need to change it 😅